exercise/calculate_next_element.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module contains a code for exercises 10-2 related to:
+Think Python, 2nd Edition
+Chapter 10: Lists
+
+This exercise requires entering a list and printing a 
+new list with the i + n = i + (i + 1) + (i+2) +...+ (i+n-1)
+"""
+
+list_a = [5, 7, 9]
```

Comment:
Function names, variable names must be meaningful.
Please double-check for all.